assets/scss/modules/_forms.scss

```diff
@@ -337,6 +337,10 @@ input[type="tel"] {
   background-repeat: no-repeat;
   font-family: $dp-font-family-proximanova;
 
+  a {
```

Comment:
Weren't we avoing general styles over elements? Just in case that we want to have another `a` without underline on this same form.

Reply:
It is `.form-message a`, I mean, it is not a style in a general element.

![image](https://user-images.githubusercontent.com/1157864/59519427-36fc6b80-8e9e-11e9-927e-a14df7f194cd.png)

Reply:
You are right! My mistake, thanks for the explanation.